solutions/loading-web-fonts/pages/index.tsx

```diff
@@ -61,23 +77,33 @@ function Home() {
           UI.
         </Text>
         <Text>
-          One possible solution is having an adjusted fallback font that tries
-          to match the used space of our custom font to reduce the layout shift.
-          You can use{' '}
+          One solution is having an adjusted fallback font that tries to match
+          the used space of our custom font to reduce the layout shift. Next.js
+          automatically adds a fallback font that matches your custom font, and
+          you can also do it manually with{' '}
           <Link
             href="https://www.industrialempathy.com/perfect-ish-font-fallback/?font=Inter"
             target="_blank"
           >
             this tool
-          </Link>{' '}
-          to find which fallback font might match your custom font and set the
-          correspondant <Code>line-height</Code> to it so it adjust accordingly.
+          </Link>
+          .
         </Text>
         <Text>
           Here we can see the difference that using an adjusted fallback font
-          makes:
+          makes, in a comparison between the <Code>Inter</Code> font (left) and
+          a custom
+          <Code>Inter-fallback</Code> font (right) which is <Code>Arial</Code>{' '}
+          with{' '}
+          <Link
+            href="https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/size-adjust"
+            target="_blank"
+          >
+            <Code>size-adjust</Code>
+          </Link>{' '}
+          set to <b>107%</b>:
         </Text>
-        <FontMixer fonts={['Inter', 'Inter-fallback']}>
+        <FontMixer fonts={['var(--inter-font)', 'Inter-fallback']}>
```

Comment:
Not sure whether the fallback works like that, or is it hard coded in the CSS?

Reply:
There's a hard coded fallback font in a CSS file, the `FontMixer` component shows a comparison of two fonts, and because I don't know the name of the generated fallback font we use a predefined one here, but that's only used internally for the purposes of the example.